staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -300,15 +309,38 @@ func NewCacherFromConfig(config Config) *Cacher {
 		panic("storage codec doesn't seem to match given type: " + err.Error())
 	}
 
+	var indexedTrigger *indexedTriggerFunc
+	if config.IndexerFuncs != nil {
+		// For now, we don't support multiple trigger functions defined
+		// for a given resource.
+		if len(config.IndexerFuncs) > 1 {
+			// community changed NewCacherFromConfig signature in
+			// https://github.com/kubernetes/kubernetes/commit/ee13be28845e8c55f6e2a2207ca4f46cd67fdb4e
+			// Propagate error from creating cacher and storage decorators up
+			// Jul 15, 2019
+
+			//return nil, fmt.Errorf("cacher %s doesn't support more than one IndexerFuncs: ", reflect.TypeOf(obj).String())
+			return nil
```

Comment:
if we only support 1 indexFunc, this should return error instead of nil.

Reply:
Community changed the interface of NewCacherFromConfig to return surface errors but Arktos does not support it now. Keep the interface now to avoid massive changes.